src/mono/wasm/debugger/tests/debugger-test/wasm-page-without-assets.html

```diff
@@ -6,6 +6,7 @@
 	<script type='text/javascript'>
 		var App = {
 			init: function () {
+				console.debug ("mono_wasm_runtime_ready", "fe00e07a-5519-4dfe-b35a-f867dbaf2e28");
```

Comment:
Is something depending on this behavior? if so we should fix it?

Reply:
The firefox debugger tests.
But @radical changed other things and now this PR is old, I need to merge main and also I'm planning to fix this:
https://github.com/dotnet/runtime/issues/73651 in the same PR, then we can totally remove this message probably.